packages/server/src/github/trigger.js

```diff
@@ -0,0 +1,62 @@
+const assert = require('assert');
+const _ = require('lodash');
+
+const { API } = require('./api');
+const { GlobalSettingsData } = require('../data/global-settings-data');
+
+class Trigger {
+  /**
+   * @param {object} webhook
+   */
+  constructor(webhook, xHubSignature) {
+    this.webhook = webhook;
+    this.xHubSignature = xHubSignature;
+    this.api = new API();
+    this.globalSetting = new GlobalSettingsData();
+
+    assert(_.isObject(webhook), 'WebHook should be an object');
+    assert(_.has(webhook, 'pull_request.merged'), 'WebHook is not in format #1');
```

Comment:
I don't like the ``format #1`` phrase. Maybe something like,
``Webhook does not have required pullrequest.merged attribute. This may be because it's in the wrong format.``

Reply:
Sure, thanks, it was delivered to later thinking.

Reply:
Fixed.